indico/modules/legal/controllers.py

```diff
@@ -38,3 +40,31 @@ def _process(self):
         if url:
             return redirect(url)
         return WPDisplayPrivacyPolicy.render_template('privacy.html', content=legal_settings.get('privacy_policy'))
+
+
+class RHAcceptTerms(RH):
+    def _process(self):
+        terms_date = legal_settings.get('terms_effective_date')
+        user_accepted = (
+            session.user.accepted_tos_dt
+            and session.user.accepted_tos_dt >= terms_date
+        )
+        if not request.args.get('preview', False) and (
```

Comment:
what's this `preview` arg? it doesn't seem to be used anywhere

Reply:
Ah I believe used this to preview the terms accept without having to fiddle around with dates, so more of a developer tool. Happy to remove this.